meson_options.txt

```diff
@@ -1,3 +1,4 @@
 option('xwayland', type: 'feature', value: 'auto', description: 'Enable support for X11 applications')
 option('systemd', type: 'feature', value: 'auto', description: 'Enable systemd integration')
 option('legacy_renderer', type: 'feature', value: 'disabled', description: 'Enable legacy renderer')
+option('pluginenv', type: 'boolean', value: false, description: 'Generate protocol headers and exit')
```

Comment:
Hmm. Why not produce both as part of the same installation process? Do people typically want only the headers, but not to install hyprland itself?

Reply:
No, I think it's fine if we provide both when installing.